atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java

```diff
@@ -56,7 +69,31 @@ public void register(Object resource) {
                     }
                 },
                 true);
-        return new AtlasDbServicesImpl(tm);
+
+        KeyValueService kvs = tm.getKeyValueService();
+        TransactionService transactionService = TransactionServices.createTransactionService(kvs);
+        SweepStrategyManager sweepStrategyManager = SweepStrategyManagers.createDefault(kvs);
+        CleanupFollower follower = CleanupFollower.create(ImmutableSet.<Schema>of());;
+        SweepTaskRunner sweepRunner = new SweepTaskRunnerImpl(
+                tm,
+                kvs,
+                new Supplier<Long>() {
+                    @Override
+                    public Long get() {
+                        return tm.getUnreadableTimestamp();
+                    }
+                },
+                new Supplier<Long>() {
+                    @Override
+                    public Long get() {
+                        return tm.getImmutableTimestamp();
+                    }
+                },
```

Comment:
Yeah... I was having similar frustration.  I think we can bump the version safely without breaking things that depend on atlas internally.  I'll do that in a separate commit and merge that in first.